src/main/java/Duke.java

```diff
@@ -1,10 +1,130 @@
+    public static final String MARKUNDONE = "unmark ";
+    public static final String TODO = "todo ";
+    public static final String DEADLINE = "deadline ";
+    public static final String EVENT = "event ";
+    public static final String LIST = "list";
+    public static final String BYE = "bye";
+    public static final String AT = "/at ";
+    public static final String BY = "/by ";
+    public static int itemCount = 0;
+
+    private static void printDashLine() {
+        System.out.println("__________________________________________________ \n");
+    }
+
+    private static void markDone (Task[] inputs, String input){
+        System.out.println("Okiii... This task has been marked as done");
+        int taskNumber = Integer.parseInt(input.substring(input.indexOf(MARKDONE) + MARKDONE.length()));
+        if (taskNumber < itemCount) {
+            inputs[taskNumber].markAsDone();
+            System.out.println(inputs[taskNumber].description);
+        } else {
+            System.out.println("Sorryyy!!! The index is out of bound!");
+        }
+        printDashLine();
+    }
+
+    private static void markUnDone (Task[] inputs, String input){
+        System.out.println("Okiii... This task has been marked as not done yet");
+        int taskNumber = Integer.parseInt(input.substring(input.indexOf(MARKUNDONE) + MARKUNDONE.length()));
+        if (taskNumber < itemCount) {
+            inputs[taskNumber].markAsUndone();
+            System.out.println(inputs[taskNumber].description);
+        } else {
+            System.out.println("Sorryyy!!! The index is out of bound!");
+        }
+
+        printDashLine();
+    }
+
+    private static void listTasks (Task[] inputs){
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < itemCount; i++) {
+            System.out.println(Integer.toString(i + 1) + " " + inputs[i].toString());
+        }
+        printDashLine();
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo = "  ____                 _     \n" +
+                " |  _ \\ ___  __ _  ___| |__  \n" +
+                " | |_) / _ \\/ _` |/ __| '_ \\ \n" +
+                " |  __/  __/ (_| | (__| | | |\n" +
+                " |_|   \\___|\\__,_|\\___|_| |_|\n" +
+                "                             ";
+        System.out.println(logo);
+
+        String intro = "__________________________________________________ \n"
+                + "  Hello! I'm Peach  \n"
+                + "  What can I do for you? \n"
+                + "__________________________________________________ \n";
+
+        System.out.println(intro);
+        Task[] inputs = new Task[100];
```

Comment:
Input and inputs sound confusing when you are not storing the input directly into inputs